content/community/webinars.md

```diff
@@ -23,7 +23,7 @@ g3Previous:
 
 <section class="g3-bg__mint">
   <div class="g3-outer-wrapper g3-flex-content g3-flex-content__reverse">
-    <div class="g3-col__65 g3-flex-content g3-space__padding-md-top g3-mb-space__padding-md-bottom">
+    <div class="g3-col__65 g3-flex-content g3-space__margin-md-top-bottom">
```

Comment:
There's a white gap at the top of the screen if you go to mobile version, could you add `g3-mb-space__padding-lg-top` class here? sorry for the crazy css classes 😛